src/datepicker/datepicker.component.ts

```diff
@@ -238,6 +239,11 @@ export class DatePicker implements OnDestroy, OnChanges, AfterViewChecked, After
 		}
 	}
 
+	ngAfterContentInit() {
+		(languages.default.en.weekdays.shorthand as string[])
+			= languages.default.en.weekdays.shorthand.map(day => day === "Thu" || day ===  "T" ? "Th" : day.charAt(0));
```

Comment:
Just curious how `carbon-react` handles this? This works, but I wonder if there's a simpler way? Also maybe make the map function a bit more explicit - that's a pretty dense oneliner, so it's kinda tricky to parse.

Reply:
They handle it in a similar way https://github.com/carbon-design-system/carbon/blob/v10.11.0/packages/components/src/components/date-picker/date-picker.js#L32-L39

Reply:
I'll make the map a little less dense though 😅